pyanaconda/modules/timezone/timezone.py

```diff
@@ -259,3 +293,28 @@ def geolocation_result(self):
         :return GeolocationData: result of the lookup, empty if not ready yet
         """
         return self._geoloc_result
+
+    def get_system_date_time(self):
+        """Get system time as a ISO 8601 formatted string.
+
+        :return: system time as ISO 8601 formatted string
+        :rtype: str
+        """
+        # FIXME: remove
+        log.debug("AAA TIME - CURRENT TIMEZONE")
+        log.debug(self._timezone)
+        # FIXME: check if this is correct to do ?
+        if self._timezone is None:
+            timezone = "America/New_York"
+        else:
+            timezone = self._timezone
+        task = GetSystemDateTimeTask(get_timezone(timezone))
+        return task.run()
+
+    def set_system_date_time(self, date_time_spec):
+        """Set system time based on a ISO 8601 formatted string.
+
+        :param str date_time_spec: ISO 8601 time specification to use
+        """
+        task = SetSystemDateTimeTask(date_time_spec=date_time_spec, timezone=self._timezone)
+        task.run()
```

Comment:
The setting might take longer, but I think it's fine without the task too. Even so, this would not be the way to do it: You just run the task here synchronously, so there is no benefit.